sdk/sphere/azure-mgmt-sphere/_meta.json

```diff
@@ -6,6 +6,6 @@
     "@autorest/python@6.6.0",
     "@autorest/modelerfour@4.26.2"
   ],
-  "autorest_command": "autorest specification/sphere/resource-manager/readme.md --generate-sample=True --include-x-ms-examples-original-file=True --python --python-sdks-folder=/home/vsts/work/1/azure-sdk-for-python/sdk --use=@autorest/python@6.6.0 --use=@autorest/modelerfour@4.26.2 --version=3.9.6 --version-tolerant=False",
+  "autorest_command": "autorest specification/sphere/resource-manager/readme.md --generate-sample=True --include-x-ms-examples-original-file=True --python --python-sdks-folder=/home/vsts/work/1/azure-sdk-for-python/sdk --use=@autorest/python@6.6.0 --use=@autorest/modelerfour@4.26.2 --version=3.9.6 --models-mode=msrest --combine-operation-files=false",
   "readme": "specification/sphere/resource-manager/readme.md"
```

Comment:
It is convenient for us to compare code change with `--combine-operation-files=false`, which has default value `true`.